themes/V3/Blank/snippets.js

```diff
@@ -21,6 +21,11 @@ module.exports = [
 				icon : 'fas fa-file-alt',
 				gen  : '\n\\page\n'
 			},
+			{
+				name : 'Page Break with Footer',
+				icon : 'fas fa-file',
+				gen  : '\n{{footnote PART 1 | SECTION NAME}}\n{{pageNumber,auto}}\n\n\\page\n'
+			},
```

Comment:
This new snippet seems a little redundant, given that we already have `New Page` and `Page Number` which will do the same thing.

How about just this set of snippets:

- New Page
- Page Number (remove the footer part of this snippet)
- Auto-incrementing page number (remove the footer part)
- Footer (new in this PR)

Then we don't have so much overlap in these snippets.

Reply:
I'll shift these to the Blank theme and shuffle them into better positions in the next commit - hopefully in the next couple of hours.

Reply:
Done!